.github/workflows/lint.yml

```
@@ -1,17 +1,20 @@
 name: Lint
-on: [ pull_request ]
+on: [pull_request]
+
+permissions:
+  contents: read
 
 jobs:
   golangci:
     name: lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
+      - uses: actions/setup-go@v4
         with:
-          go-version: 1.19
-          check-latest: true
+          go-version: 1.20.6
+          cache: false
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.2.0
+        uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.50.1
+          version: v1.53.3
```

```suggestion
          version: v1.53
```

then we dont need to update for patch releases

---

Changed